pandas/tests/extension/test_categorical.py

```diff
@@ -207,6 +207,20 @@ def test_cast_nan_to_int(self, cls, values):
         with pytest.raises((ValueError, TypeError), match=msg):
             s.astype(int)
 
+    @pytest.mark.parametrize(
+        "expected",
+        [
+            pd.Series([pd.Period("2019"), pd.Period("2020")], dtype="period[A-DEC]"),
```

Comment:
can you add in a datetime w/tz, and a timedelta case

Reply:
Added a datetime test but wasn't sure how to get a timedelta that'd be considered an extension dtype

Reply:
just use dtype='timedelta64'